extensions/windows-event-log/ConsumeWindowsEventLog.cpp

```diff
@@ -712,8 +712,8 @@ void ConsumeWindowsEventLog::putEventRenderFlowFileToSession(const EventRender&
       session.write(flowFile, &wc);
     }
     session.putAttribute(flowFile, core::SpecialFlowAttribute::MIME_TYPE, mimeType);
-    session.putAttribute(flowFile, "Timezone name", timezone_name_);
-    session.putAttribute(flowFile, "Timezone offset", timezone_offset_);
+    session.putAttribute(flowFile, "timezone.name", timezone_name_);
+    session.putAttribute(flowFile, "timezone.offset", timezone_offset_);
```

Comment:
IMO it is the responsibility of InvokeHTTP to adjust the attribute keys to create a valid HTTP header, CWEL should not concern itself with how its output might be used

Reply:
I thought the same at first, but then most other attributes have this format of dot separated lowercase words, so it's improving consistency anyway.

Reply:
I think this change has merit, but I also think that as it currently stands, it does not address the root cause of the issue, note that we also append all attributes of the "Data" xml node as flowfile attributes (granted attribute names probably cannot contain spaces)

Reply:
You are right that that CWEL should not concern itself with other processors, but as windows event logs forwarded through InvokeHTTP is a use case that is being implemented by one of our users this should be fixed, also as Marton mentioned our attribute names should be consistent. The root cause is that the HTTP headers do not allow these kinds of attributes and we cannot really do much about that other than adapting our code for this scenario. If we find other instances like this I think they should also be corrected, but I don't think we have any other known issues of this.